index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <script src="https://use.fontawesome.com/382f947f1c.js"></script>
+  <link href="https://fonts.googleapis.com/css?family=Playfair+Display:400,400i,700|Raleway:300,500,700" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="CSS/style.css">
+  <title>Aloha Homepage</title>
+</head>
+<body>
+    <header>
```

Comment:
Add container class (max-width: 1240px) to your CSS, and then add throughout mark-up where needed.